packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -129,18 +146,19 @@ const AccountWishlist = () => {
                 />
             )}
 
-            {wishlist.hasDetail &&
-                !wishlist.isEmpty &&
-                wishlist.items.map((item) => (
+            {!isPageLoading &&
+                wishListItems &&
+                wishListItems.map((item) => (
                     <ProductItem
```

Comment:
The code in `develop` removes the item from the wishlist when the shopper decrements the product quantity < 1, when taking either of these two actions in the QuantityPicker:
1. Typing the number '0' in the quantity Input field.
2. Pressing the '-' button when the quantity is 1.